package.json

```diff
@@ -21,32 +21,32 @@
     "@account-abstraction/contracts": "^0.5.0",
     "@gnosis.pm/safe-contracts": "^1.3.0",
     "@nomicfoundation/hardhat-chai-matchers": "^1.0.6",
-    "@nomicfoundation/hardhat-network-helpers": "^1.0.8",
+    "@nomicfoundation/hardhat-network-helpers": "^1.0.9",
     "@nomicfoundation/hardhat-toolbox": "^2.0.2",
     "@nomiclabs/hardhat-ethers": "^2.2.3",
     "@nomiclabs/hardhat-etherscan": "^3.1.7",
-    "@typechain/ethers-v5": "^10.2.0",
-    "@typechain/hardhat": "^6.1.5",
-    "@types/mocha": "^10.0.1",
-    "chai": "^4.3.7",
-    "eslint": "^8.39.0",
+    "@typechain/ethers-v5": "^11.1.2",
+    "@typechain/hardhat": "^9.1.0",
+    "@types/mocha": "^10.0.3",
+    "chai": "^4.3.10",
+    "eslint": "^8.52.0",
     "ethers": "^5.7.2",
-    "hardhat": "^2.14.0",
-    "hardhat-contract-sizer": "^2.8.0",
-    "hardhat-deploy": "^0.11.26",
+    "hardhat": "^2.18.2",
```

Comment:
This is required in order to be able to deploy on networks that don't yet have `PUSH0` instructions available.